.ci/docker/check-docker-compose-services.sh

```diff
@@ -15,5 +17,7 @@ is_healthy() {
     fi
 }
 
-while ! is_healthy trial-config-service; do sleep 5; done
-while ! is_healthy practitioner-service; do sleep 5; done
\ No newline at end of file
+while ! is_healthy config-server; do sleep 10; done
+while ! is_healthy site-service; do sleep 10; done
+while ! is_healthy practitioner-service; do sleep 10; done
+while ! is_healthy role-service; do sleep 10; done
```

Comment:
add echo "Services running..."  at the eof?

Reply:
Added a success message